src/app/src/mainwindow.h

```diff
@@ -214,7 +217,9 @@ class MainWindow : public QMainWindow {
         bool ledOFF();
         bool ledSetVoltage(double voltage);
         bool available_space_in_default_drive( double fps,double duration);
+        //acquire helper function
         void acquire(bool saveToDisk);
+        double calc_max_frame_rate(int p1,int p2,  int16_t spdtable_index,double m_line_times [4],uint16_t spdtable);
```

Comment:
can you use camelCase for private function names instead so it matches the style of the test of the codebase